unified-runtime/source/loader/layers/sanitizer/sanitizer_common/sanitizer_utils.hpp

```diff
@@ -19,6 +19,19 @@
 #include <string>
 #include <vector>
 
+// Trace an internal UR call, storing the result to Res and break instead of
+// return when not success
+#define UR_CALL_BREAK(Res, Call)                                               \
+  {                                                                            \
+    if (PrintTrace)                                                            \
+      UR_LOG(QUIET, "UR ---> {}", #Call);                                      \
+    Res = (Call);                                                              \
+    if (PrintTrace)                                                            \
+      UR_LOG(QUIET, "UR <--- {}({})", #Call, getUrResultString(Res));          \
```

Comment:
I just merged https://github.com/intel/llvm/pull/19055 which removes `getUrResultString()` because it is not necessary, the logger supports pretty printing `ur_result_t`s already.

```suggestion
      UR_LOG(QUIET, "UR <--- {}({})", #Call, Res);                             \
```